F5 Networks/f5-big-ip/ingest/parser.yml

```
@@ -280,7 +280,7 @@ stages:
           observer.ip: "{{parse_cef_event.message.dvc}}"
           observer.hostname: "{{parse_cef_event.message.dvchost}}"
           observer.type: "firewall"
-          host.name: "{{parse_cef_event.message.dvchost}}"
+          host.name: "{{parse_cef_event.message.dvchost or parse_cef_event.message.host}}"
           host.hostname: "{{parse_cef_event.message.dvchost}}"
 
```

I checked the value of 'host'.
For HTTP request, this value seems to match the `HOST` header in the request. So, I highly recommend moving this value to `url.domain` for `HTTP Request`.

```suggestion
          host.name: "{{parse_cef_event.message.dvchost}}"
          host.hostname: "{{parse_cef_event.message.dvchost}}"

      - set:
          url.domain: "{{parse_cef_event.message.host}}"
        filter: "{{parse_cef_event.message.act == 'HTTP Request'}}"
```